web/src/api/getZone.ts

```diff
@@ -21,73 +18,56 @@ import {
   TIME_RANGE_TO_BACKEND_PATH,
 } from './helpers';
 
-const getZone = async (
+const getState = async (
   timeRange: TimeRange,
-  zoneId: string,
-  is1HourAppDelay: boolean,
   targetDatetime?: string
-): Promise<ZoneDetails> => {
-  invariant(zoneId, 'Zone ID is required');
-
+): Promise<GridState> => {
   const shouldQueryHistorical =
     targetDatetime &&
     isValidDate(targetDatetime) &&
     isValidHistoricalTimeRange(timeRange);
 
   const path: URL = new URL(
-    `v10/details/${TIME_RANGE_TO_BACKEND_PATH[timeRange]}/${zoneId}${getParameters(
+    `v10/state/${TIME_RANGE_TO_BACKEND_PATH[timeRange]}${getParameters(
```

Comment:
Why is the URL updated here? It will completely break the app as it's not the same data that is being fetched.